spec/integration/projects_spec.rb

```diff
@@ -49,7 +49,6 @@
       expect(project.average_price_per_tonne_cents_usd)
         .to be_an_instance_of(Integer)
       expect(project.remaining_mass_g).to be_an_instance_of(Integer)
-      expect(project.state).to be_an_instance_of(String)
```

Comment:
The underlying data has changed and the project being picked up doesn't have a state (now that it is optional)

Reply:
no test added for verifier though?